lib/src/HttpUtils.cc

```diff
@@ -607,11 +607,15 @@ FileType parseFileType(const string_view &fileExtension)
 {
     // https://en.wikipedia.org/wiki/List_of_file_formats
     static const std::unordered_map<string_view, FileType> map_{
+        {"", FT_UNKNOWN},
         {"html", FT_DOCUMENT},
         {"docx", FT_DOCUMENT},
         {"zip", FT_ARCHIVE},
         {"rar", FT_ARCHIVE},
         {"xz", FT_ARCHIVE},
+        {"7z", FT_ARCHIVE},
+        {"tgz", FT_ARCHIVE},
+        {"gz", FT_ARCHIVE},
```

Comment:
I think we should add extensions with more than one dot, for example .tar.gz, do you think it would be a good idea to implement this?

Reply:
Come to think of it, this will make it very ambiguous and it will be very difficult to distinguish whether it is really a 2-point file or not. But even so, it's still possible, every time we would fetch the extension we would have to go to our list of existing "FileTypes" and compare if any of them match ours and so on, problem solved.

Reply:
I think the tar.gz just is the gz, tar is the internal nested type of the file, if we add tar.gz to the list, there are lots of extensions that should be added to the list, such as `txt.gz`, `cpp.gz`, `tar.7z` and so on.

Reply:
Hmmm, so it would be better to forget this part?

Reply:
Yes, I think getting the last part is okay.